include/networkit/auxiliary/BucketPQ.hpp

```diff
@@ -40,7 +40,9 @@ class BucketPQ: public PrioQueue<int64_t, index> {
     count numElems;							// number of elements stored
     int64_t offset;							// offset from minAdmissibleKeys to 0
 
-private:
+    Bucket dummyBucket = Bucket{};
+    const Bucket::iterator invalidPtr;
```

Comment:
These can probably be static constexpr (?).

Reply:
`std::list` and `std::list::iterator` are not literals so I am not sure if that's possible (if I make them static constexpr I get errors).

Reply:
What if they are static but not constexpr? Basically, we should avoid per-PQ dummy objects that waste a few bytes but do absolutely nothing.

Reply:
Yes that works, now `dummyBucket` is static.